clicommand/agent_start.go

```diff
@@ -939,6 +950,30 @@ var AgentStartCommand = cli.Command{
 			l.Info("Agents will disconnect after %d seconds of inactivity", agentConf.DisconnectAfterIdleTimeout)
 		}
 
+		if len(cfg.AllowedRepositories) > 0 {
+			agentConf.AllowedRepositories = []*regexp.Regexp{}
+			l.Info("Repositories allow list is enabled.")
```

Comment:
i think we can probably get rid of this log, it's not saying much that's useful, and we don't really have a pattern of logs when features are enabled

```suggestion
```